python-sdk/tests/benchmark/wait_for_completion.sh

```diff
@@ -10,7 +10,7 @@ while [[ $complete_status -ne 0 ]] && [[ $failed_status -ne 0 ]]; do
   complete_status=$?
 done
 
-kubectl delete job benchmark-"${GIT_HASH}" -n benchmark
+#kubectl delete job benchmark-"${GIT_HASH}" -n benchmark
```

Comment:
Any reason why this line is commented?

Reply:
Temp change. We need to see the logs and pods will be deleted post-run if this is not added.